modules/tensor_mechanics/test/tests/visco/gen_kv_driving.i

```diff
@@ -1,3 +1,9 @@
+# represents a unique Maxwell module with E = 10GPa and eta = 10 days with an imposed eigenstrain alpha = 0.001
+# the specimen is free (sigma = 0) but the eigenstrain contributes to the creep deformation
```

Comment:
Is that really right?  It seems like the elastic strain would be zero, so you'd get no creep strain.

Reply:
This test (and its companion gen_maxwell_driving) represents materials in which the creep strain is driven by the stress + the equivalent stress imposed by the eigenstrain (the driving_eigenstrain variable). This technique has been used by some authors to represent drying creep in concrete (additional creep strain induced by drying shrinkage).

In this test, the elastic strain is indeed 0, but the driving eigenstrain is added to the elastic strain when performing the update of the viscous strains. So there is a creep strain (which is the same creep strain as you would have under an elastic strain equal to the driving eigenstrain), and the total strain (epsilon in the comment) is therefore different from the eigenstrain + the creep strain derived from the eigenstrain. I checked that in this condition, MOOSE indeed gives an elastic strain equal to 0 (1e-16 on my machine).

I can modify the comment to clarify that if needed.

Reply:
OK, that makes sense.  While you're making that other change, if you could explain that a little bit more in the test input file, that would be good. Other than that, I think this is good to go.